src/server/handlers.rs

```diff
@@ -102,6 +103,15 @@ pub(crate) fn handle_test(
             message: "INFO: Test completed successfully".to_string(),
         },
     );
+    global_state.send_notification::<bsp_types::notifications::TaskFinish>(
+        bsp_types::notifications::TaskFinishParams {
```

Comment:
Ma sens, ale czy to wcześniejsze notification  nie powinno być też`TaskStart` zamiast `LogMessage`?

Reply:
Zgodnie z dokumentacja protokołu TaskStart może (nie musi) zostać użyte a TaskFinish powinno zostać użyte. LogMessage służy do wypisywania w konsoli użytkownika i używamy tego również przy requestach run i build.

Reply:
dobra faktycznie, my bad